### PR TITLE
Allow to replace the console used by printit

### PR DESCRIPTION
**Pull request by nono**
It can be used to send logs to a file for example

### CODE REVIEW

File: main.coffee [hunk 1]
```diff
@@ -1,6 +1,7 @@
 dateFormat = require './date-format'
 
-module.exports = (options) -> new Logger options
+create = (options) -> new Logger options
```
Comment (frankrousseau) [inline main.coffee]: The name `create` is misleading. We don't understand why a console is attached to it.

Comment (nono) [inline main.coffee]: Yes, I'm not very inspired. Do you have a suggestion?

Comment (nono) [inline main.coffee]: `printit`?

Comment (frankrousseau) [inline main.coffee]: Yes printit

File: README.md [hunk 2]
```diff
@@ -41,3 +41,24 @@ log.error("Print this with a red error label and my app prefix");
...
+var out = fs.createWriteStream('log.txt');
+printit.console = new Console(out, out);
+
+var log = printit({
+  prefix: 'my app',
+  date: true
+});
+
+log.info("Print this with a blue info label and my app prefix and date");
+```
```
Comment (kosssi) [inline README.md]: maybe you can show the log.txt output?

**Pull request merged**